examples/c/ex_file_system.c

```diff
@@ -31,6 +31,12 @@
 #include <errno.h>
 #include <stdlib.h>
 #include <string.h>
+#ifndef _WIN32
+#include <unistd.h>
```

Comment:
@daveh86 Yes - please make that change. Maybe copy the clause from ``ex_scope.c``